### ISSUE TITLE
Enhanced Font Preview in Settings > Appearance

### ISSUE THREAD

**Issue by intherightdirection** [microsoft/terminal#17519]
# Description of the new feature/enhancement

**Comment by zadjii-msft** [comment]
crazier idea: what if we just let you type in that preview? So you could type whatever you wanted?

**Comment by DHowett** [comment]
That would be a focus trap - the Terminal control aggressively asserts focus. We should be careful heading in that direction.

**Comment by zadjii-msft** [comment]
bad idea: add a fake `IKeybinding` to that `TermControl` which is bound to <kbd>tab</kbd> and handled in the settings UI and use that to manually escape to the next control

**Comment by DHowett** [comment]
worse idea: what if we just put a text box down there, whose content is copied into the preview control on a line below the prompt
